packages/yoastseo/src/worker/AnalysisWebWorker.js

```diff
@@ -6,27 +6,27 @@ import { forEach, has, includes, isEmpty, isNull, isObject, isString, isUndefine
 import { getLogger } from "loglevel";
 
 // YoastSEO.js dependencies.
```

Comment:
This comment is a little bit outdated 😅 Could you change the order of the imports so all assessors are together? Thanks!